kubernetes_vars.yml

```diff
@@ -0,0 +1,76 @@
+# ansible remote user account
+# ansible_ssh_user: centos
+
+# ansible groups for service configuration
+etcd_group_name: role=control
```

Comment:
can we add this variable to the roles/etcd/defaults/main.yml so that etcd can be included without kubernetes if needed?